initials.c

```diff
@@ -0,0 +1,12 @@
+/* Ashank Kumar */
+#include <stdio.h>
```

Comment:
Perfect! My only suggestion is to include a small description of what this program does in your comment at the top!